lib/ngComponents.js

```diff
@@ -0,0 +1 @@
+/*** AUTO-GENERATED BY GULP ***/
```

Comment:
But if we do this then we will keep committing the changes whenever we work locally. I think a better option would be to output all the generated files in a folder, say "build/" or ".cache/" which would be in gitignore. If necessary we could store "template" files in git which can be copied and edited by gulp.